eo/src/eoRanking.h

```diff
@@ -40,73 +40,175 @@ template <class EOT>
 class eoRanking : public eoPerf2Worth<EOT> // false: do not cache fitness
 {
 public:
-
-    using eoPerf2Worth<EOT>::value;
+  using eoPerf2Worth<EOT>::value;
 
   /* Ctor:
    @param _p selective pressure (in (1,2]
    @param _e exponent (1 == linear)
   */
-  eoRanking(double _p=2.0, double _e=1.0):
-    pressure(_p), exponent(_e) {}
+  eoRanking(double _p = 2.0, double _e = 1.0) : pressure(_p), exponent(_e) {}
```

Comment:
May I ask you to use this occasion to improve a little bit of the old code ? :-)
```suggestion
  eoRanking(double _p = 2.0, double _e = 1.0) : pressure(_p), exponent(_e) {
      assert(1 < pressure and exponent <= 2);
  }
```